buildpipeline/Core-Setup-PortableLinux-x64.json

```diff
@@ -97,8 +97,180 @@
     },
     {
       "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Copy built Portable binaries to staging directory",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(Build.SourcesDirectory)/artifacts/linux-x64/intermediate/sharedFrameworkPublish",
```

Comment:
Because 'git clean -xdf' is called for each step of building docker image. Git clean is required and removes the artifacts folder. Copying to staging directory to preserves the folder.